script/core/monitors/MarketHealth.s.sol

```diff
@@ -0,0 +1,156 @@
+// SPDX-License-Identifier:	AGPL-3.0
+pragma solidity ^0.8.13;
+
+import {Deployer} from "mgv_script/lib/Deployer.sol";
+import {Test2, console2 as console} from "mgv_lib/Test2.sol";
+import {UpdateMarket} from "mgv_script/periphery/UpdateMarket.s.sol";
+import {MgvReader, VolumeData, IMangrove} from "mgv_src/periphery/MgvReader.sol";
+import {IERC20} from "mgv_src/IERC20.sol";
+import {MgvStructs} from "mgv_src/MgvLib.sol";
+
+/**
+ * @notice Script to obtain data about a given mangrove half book. Data is outputted to terminal as space separated values.
+ * @notice retrieved data are:
+ *  "tot_bounty": collected bounty
+ *  "tot_failures": total failures
+ *  "tot_gas_fail": gas consumed by failing offer
+ *  "tot_gas_req": gas required by makers
+ *  "tot_gas_used": gas consumed for the market order
+ *  "tot_successes": number of successful offers
+ *  "volume_received": volume received (in display units of outbound token)
+ *  "volume_sent": volume sent (in display units of inbound token)
+ *
+ * @notice environment variables that control the script are:
+ * @param VOLUME the amount of outbound token that are required from the market
+ * @param TKN_IN the inbound token (token that are sent by the tester)
+ * @param TKN_OUT the outbound token (token that are required by the tester)
+ */
+/**
+ * Usage: testing status of buy orders on the WMATIC,USDT market for volumes of up to 100,000 WMATIC (display units)
+ *  VOLUME=$(cast ff 18 100000) TKN_IN=USDT TKN_OUT=WMATIC \
+ *  forge script --fork-url mumbai MarketHealth
+ */
+contract MarketHealth is Test2, Deployer {
+  // needed if some offer fail, in order to receive bounty
+  uint[] internal failingIds;
+
+  receive() external payable {}
+
+  function run() public {
+    IERC20 inbTkn = IERC20(envAddressOrName("TKN_IN"));
+    // dealing hopefully enough inbound token to execute a market order
+    deal(address(inbTkn), address(this), 10_000_000 * 10 ** inbTkn.decimals());
```

Comment:
Would `type(uint).max` work?

Reply:
Probably would as well but since I have the required value I might as well use it.